src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/pages/IdeaFrame.kt

```diff
@@ -0,0 +1,67 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.data.RemoteComponent
+import com.intellij.remoterobot.fixtures.CommonContainerFixture
+import com.intellij.remoterobot.fixtures.ComponentFixture
+import com.intellij.remoterobot.fixtures.DefaultXpath
```

Comment:
So, for every IntelliJ window you have to define any element you use?!

Reply:
Yes, you need to make a separate Frame for every window that appears. From what I know, this is not necessary, but it helps a lot, because it allows you to separate your logic.